pointers/pointers.c

```diff
@@ -21,7 +24,8 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+  int len = strlen(*str);
+  printf("%d <<< length \n", len);
```

Comment:
You can take advantage of that null character and look for a character in this string like looking for a node in a linked list.